config/application.rb

```diff
@@ -58,6 +58,17 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('app', 'validators')
 
+    # :nocov:
+    if Rails.env.production?
+      config.cache_config_file = '/var/lib/rmt/rmt-cache-trim.sh'
+      config.repo_cache_dir = '/run/rmt/cache/repository'
+      cache_config_content = %(REPOSITORY_CLIENT_CACHE_DIRECTORY=#{config.repo_cache_dir}
+REPOSITORY_CACHE_EXPIRY_MINUTES=20
```

Comment:
Nitpick: The lack of indent here is kind of ugly.

Consider using a tilde (~) heredoc or a joined array instead; both will result in nicely indented code, but no unwantedwhitespace in the result.

```
cache_config_content = <<~EOF
  REPOSITORY_CLIENT_CACHE_DIRECTORY=#{config.repo_cache_dir}
  REPOSITORY_CACHE_EXPIRY_MINUTES=20
EOF
```

```
cache_config_content = [
  "REPOSITORY_CLIENT_CACHE_DIRECTORY=#{config.repo_cache_dir}",
  "REPOSITORY_CACHE_EXPIRY_MINUTES=20"
].join("\n")
```